os_build_stories/raspbian_9_stretch

```diff
@@ -0,0 +1,54 @@
+cmake:
+
+    sudo apt-get install cmake
+
+xsltproc:
+
+    sudo apt-get install xsltproc
+
+tidy-html5:
+
+    git clone git://github.com/htacg/tidy-html5
+    cd tidy-html5/build/cmake
+    cmake ../..
+    make
+    sudo make install
+
+curl:
+
+    sudo apt-get install libcurl4-openssl-dev
+
+libpcre:
+
+    sudo apt-get install libpcre3-dev
+
+readline:
+
+    sudo apt-get install libreadline-dev
+
+python modules:
+
+    sudo apt-get install python-yaml
+
+duktape:
+  Even though duktape-dev and duktape are present, installing them didn't get duktape.h
```

Comment:
It actually does, but it's not obvious to use with our build system.
The duktape-dev package installs duktape.c, duktape.h and duk_config.h in /usr/share/duktape; the point is that the packager thinks that duktape is meant to be compiled for every project which might want to change options in duk_config so they do not ship a prebuilt library (no -lduktape) and the header is not installed in standard path.

It would be possible to build that duktape.c and use that duktape.h in our build system if you'd like to improve on that, but for now installing the shared library as you did is the way to go.